src/hooks/useColor.ts

```diff
@@ -11,30 +11,43 @@ async function getColorFromToken(token: Token): Promise<string | null> {
     return COLOR_MAP.get(token.address)
   }
 
-  const path = `https://raw.githubusercontent.com/trisolaris-labs/tokens/master/assets/${token.address}/logo.png`
+  const paths = [
+    `https://raw.githubusercontent.com/trisolaris-labs/tokens/master/assets/${token.address}/logo.svg`,
+    `https://raw.githubusercontent.com/trisolaris-labs/tokens/master/assets/${token.address}/logo.png`
+  ]
 
-  return Vibrant.from(path)
-    .getPalette()
-    .then(palette => {
-      if (palette?.Vibrant) {
-        let detectedHex = palette.Vibrant.hex
-        let AAscore = hex(detectedHex, '#FFF')
-        while (AAscore < 3) {
-          detectedHex = shade(0.005, detectedHex)
-          AAscore = hex(detectedHex, '#FFF')
+  const getVibrant = (path: string) => {
```

Comment:
also, IMO, function declarations are better than function expressions bc of hoisting

Reply:
force of habit of using consts and declaring functions on top of the files, changed but maybe we can talk trough this one